numba/cpython/tupleobj.py

```diff
@@ -406,3 +406,8 @@ def tuple_index_impl(tup, value):
         raise ValueError("tuple.index(x): x not in tuple")
 
     return tuple_index_impl
+
+
+@overload(operator.contains, types.Tuple([]), types.Any)
```

Comment:
Providing a signature isn't yet implemented for `@overload`, you'll need to put the type checks into the function body. https://numba.readthedocs.io/en/stable/extending/overloading-guide.html is a good place to start learning about `@overload` and has examples patterns to follow. Due to the lack of guards on the types, this function is accepting any types and is most likely overriding other functions that do similar, as a result some test fail. Good news is that the fix should be quick!

Reply:
Ha, yeah, I was thinking of a different decorator when I whipped that together.  I've updated it now.